server/utils/zip.js

```diff
@@ -4,12 +4,26 @@ import JSZip from 'jszip';
 import tmp from 'tmp';
 import unescape from '../../shared/utils/unescape';
 import { Collection, Document } from '../models';
+import { getImageByKey } from './s3';
+
+const ENABLE_PRIVATE_CONTENT = process.env.ENABLE_PRIVATE_CONTENT === 'true';
+const s3KeyRegex = /!\[.*\]\(\/api\/images\.info\?key=(?<key>.*)\)/gi;
+const imgageApiRegex = /(?<=!\[.*\]\()(\/api\/images\.info\?key=)/gi;
 
 async function addToArchive(zip, documents) {
   for (const doc of documents) {
     const document = await Document.findByPk(doc.id);
+    let text = unescape(document.text);
+
+    if (ENABLE_PRIVATE_CONTENT) {
+      const imageKeys = [...text.matchAll(s3KeyRegex)].map(
```

Comment:
`matchAll` is from the es2020 spec, this is a test fail point at the moment - so I could go back to current convention to fix.

Reply:
Yea, lets use current convention here.